Assets/MixedRealityToolkit.Tests/PlayModeTests/PointerTests.cs

```diff
@@ -100,52 +100,76 @@ public IEnumerator TestLinePointers()
         {
-            var cube = GameObject.CreatePrimitive(PrimitiveType.Cube);
-            cube.AddComponent<NearInteractionGrabbable>();
-            var rightHand = new TestHand(Handedness.Right);
-            yield return rightHand.Show(Vector3.zero);
+            yield return testHand.Show(Vector3.zero);
+            cube.transform.SetPositionAndRotation(Vector3.zero, Quaternion.identity);
+            cube.transform.localScale = Vector3.one;
+
+            // Test case where pointer is inside of giant object
+            yield return testHand.MoveTo(cube.transform.position);
+            Assert.IsNotNull(myPointer);
+            Assert.IsTrue(myPointer.IsInteractionEnabled, "Pointer should be enabled because it is inside giant cube.");
+
+            // Move cube in front of camera so it is visible
+            cube.transform.Translate(Vector3.forward * 1f);
+            yield return testHand.MoveTo(cube.transform.position);
             yield return PlayModeTestUtilities.WaitForInputSystemUpdate();
+            Assert.IsTrue(myPointer.IsInteractionEnabled, $"Pointer {myPointer.PointerName} should be enabled because it is near cube and visible.");
 
-            var spherePointer = PointerUtils.GetPointer<SpherePointer>(Handedness.Right);
-            Assert.IsNotNull(spherePointer, "Right hand does not have a sphere pointer");
-            Assert.IsTrue(spherePointer.IsInteractionEnabled, "Sphere pointer should be enabled because it is near grabbable cube and visible, even if inside a giant cube.");
-            GameObject.Destroy(cube);
+            // Make cube no longer visible
+            cube.transform.Translate(Vector3.up * 10);
+            yield return testHand.MoveTo(cube.transform.position);
+            yield return PlayModeTestUtilities.WaitForInputSystemUpdate();
+            Assert.IsFalse(myPointer.IsInteractionEnabled, $"Pointer {myPointer.PointerName} should NOT be enabled because hand is near object that is not in FOV.");
+
+            // For sphere and poke pointers, test that setting 
+            if (myPointer is SpherePointer spherePointer)
+            {
+                spherePointer.IgnoreCollidersNotInFOV = false;
+                yield return PlayModeTestUtilities.WaitForInputSystemUpdate();
+                Assert.IsTrue(myPointer.IsInteractionEnabled, $"Pointer {myPointer.PointerName} should be enabled.");
+                spherePointer.IgnoreCollidersNotInFOV = true;
+            }
+            else if (myPointer is PokePointer pokePointer)
+            {
+                pokePointer.IgnoreCollidersNotInFOV = false;
+                yield return PlayModeTestUtilities.WaitForInputSystemUpdate();
+                Assert.IsTrue(myPointer.IsInteractionEnabled, $"Pointer {myPointer.PointerName} should be enabled.");
+                pokePointer.IgnoreCollidersNotInFOV = true;
+            }
```

Comment:
This would also be good to simplify with base class shared between the two

Reply:
Good point! I still would prefer to wait until we have more need, specifically when creating a ConePointer. I can create a bug for this and mention that following up from #7016 we should make a base class.

Reply:
Since you arent' creating an interface or base class, maybe add else statement here that throws error saying test helper isn't valid for this type of pointer? To help notify people who might try to use this in future